### ISSUE TITLE
Encryption unlock screen needs a better explanation

### ISSUE THREAD

**Issue by hanaral** [elementary/plymouth-theme#13]
### Problem

No one knows what cryptswap: cryptsetup is, this could even lead to users getting scared

### Solution

“Please enter password to unlock encryption” in the appropriate language. Even better, use good iconography and no text. 
Apple doesn’t even use text I believe as it’s annoying to translate things that probably don’t even have support for non Latin alphabets.

**Comment by hanaral** [comment]
@davidmhewitt My bad, wasn't sure where to put it.

> I believe this text is probably provided by something in cryptsetup itself, so modifying it may be difficult.

Surely there has to be a string that has that text somewhere right? Unless it's dynamically generated...

**Comment by davidmhewitt** [comment]
There probably is a string somewhere, but it's probably in cryptsetup which we get from upstream Ubuntu.

We prefer to carry patches for as few upstream Ubuntu packages as possible. It's especially undesirable to have to carry patches for security critical packages as we'd be slowing down the speed at which updates to them are available.

We'll see what people think though.

**Comment by cassidyjames** [comment]
Yeah I personally think I would be fine completely omitting that string if possible, or replacing it with a generic string if that's possible.

**Comment by meisenzahl** [comment]
plymouth will [fallback to "Sans 12"](https://gitlab.freedesktop.org/plymouth/plymouth/-/blob/c4ced2a2d70edea7fbb95274aa1d01d95928df1b/src/plugins/controls/label/plugin.c#L180) if no font description is provided.

But "Sans" should be resolved to "Arimo":

```
$ fc-match sans
Arimo-Regular.ttf: "Arimo" "Regular"
```

So I'm not quite sure what's happening here...

**Comment by cassidyjames** [comment]
@meisenzahl is it a matter of the font not being available from the OS? Do we need to do something special with the font to make it available?

**Comment by davidmhewitt** [comment]
It may be that we can improve on this slightly. It turns out that `cryptdata` is currently the value that the installer uses by default as the name for the LVM volume. 

This results in a string on the unlock screen saying "Please unlock disk cryptdata". We can change that default LVM volume name to something a bit less weird than `cryptdata`, but I'm not sure what, since it would have to follow the word "disk".

**Comment by cassidyjames** [comment]
@davidmhewitt hrm, yeah that's worded weird no matter what. It would be slightly less weird if we could wrap it in quotes or something, because I'm thinking:

>Please unlock disk “elementary OS”

works, but:

>Please unlock disk elementary OS

makes no sense.

**Comment by 4jNsY6fCVqZv** [comment]
For me, this issue fits into the field of a well-rounded "boot and shutdown" experience.
Related to this comment: https://github.com/elementary/plymouth-theme/issues/17#issuecomment-985477348.

**Comment by Odi-3** [comment]
If you decide to opt for the latter choice, here's the solution:
https://forum.manjaro.org/t/edit-manjaro-plymouth-file/104212/2
